src/js/media-chrome-dialog.ts

```diff
@@ -94,19 +96,26 @@ class MediaChromeDialog extends globalThis.HTMLElement {
   constructor() {
     super();
 
+    this.addEventListener('invoke', this);
+    this.addEventListener('focusout', this);
+    this.addEventListener('keydown', this);
```

Comment:
this makes it call `handleEvent` on the this object.

it can be handy so child classes can override the handleEvent method. 
this technique is used in a few different components.

Reply:
lol, how have I gone this long without knowing about `handleEvent`! 👍